### DISCUSSION TITLE
Bootloader and partition table question

### DISCUSSION THREAD

**Discussion by HarryD37** [revk/xESP32-Faikout#249] [General]
I am completely new to ESP's but if i understand correctly every ESP has a boot loader already flashed onto the device.
I am wondering if its necessary to re-flash it ?!  is the boot loader in the /release directory an upgraded one ?

**Comment by revk** [comment]
It has a rom but the normal esp idf build includes a boatload (on which I can configure various settings when building) that is flashed.

**Reply by HarryD37** [reply]
Thank you.
So Faikin doesnt work when i use the bootloader thats in rom ?

**Reply by revk** [reply]
I think the rom boot loader is just to allow the serial flashing, not the full bootloader. So no!
